ratis-server/src/main/java/org/apache/ratis/server/storage/RaftStorageImpl.java

```diff
@@ -41,6 +41,41 @@ public enum StartupOption {
     FORMAT
   }
 
+  public static Builder newBuilder() {
+    return new RaftStorageImpl.Builder();
+  }
+
+  public static class Builder {
```

Comment:
The Builder should be moved to RaftStorage so that it will be in raft-server-api.

Reply:
done